src/components/dropdown/README.md

```diff
@@ -466,7 +459,7 @@ regular form.
 <script>
   export default {
     methods: {
-      onClick () {
+      onClick() {
```

Comment:
For ES Lint settings, the space between the method name and parens is preferred when defining the method

Reply:
Currently this isn't possible with `prettier` but there is a [discussion](https://github.com/prettier/prettier/issues/3845) wether to add it or not.